src/layouts/SessionsSection.jsx

```diff
@@ -86,7 +86,14 @@ function SessionsSection() {
               }`}
               onClick={() => setActiveTab(index)}
             >
-              {tab}
+              {tab === 'Miscellaneous' ? (
+                <>
+                  <span className="hidden xs:inline">Miscellaneous</span>
+                  <span className="inline xs:hidden">Misc</span>
```

Comment:
Like this change - could I suggest bumping it up to small screen sizing. Miscellaneous doesn't look great a little before the xs breakpoint

<img width="403" height="195" alt="Screenshot 2025-09-21 at 9 19 49 PM" src="https://github.com/user-attachments/assets/919a8f80-6af9-4496-bd4b-357403cba642" />